python/pyspark/pandas/tests/indexes/test_base.py

```diff
@@ -516,19 +515,6 @@ def test_missing(self):
                 getattr(psdf.set_index("c").index, name)()
 
         # CategoricalIndex functions
```

Comment:
We should remove from this line to line 525 as well.

Reply:
Good catch! Removed.